homeassistant/components/template/light.py

```diff
@@ -378,6 +378,9 @@ def update_brightness(self):
             return
         try:
             brightness = self._level_template.async_render()
+            if brightness == "None" or brightness == "":
+                self._brightness = None
+                return
             if 0 <= int(brightness) <= 255:
```

Comment:
What happens if this raises ValueError?

Reply:
I'm confused that the `catch` sets `self._state` to None, should that be `self._brightness` ?

Reply:
Both fair questions - but not in the code I changed. Is it a rhetorical question for other contributors, or for me?...

Reply:
Perhaps because one template is invalid, so the whole state must be too.

Reply:
@balloob - how would you like to handle this?

Reply:
If you're touching this code, can you fix it? I think it's a copy&paste and adjust error and it should set brightness to None. 

 ValueError should be caught like TemplateError, but have a different message.

Reply:
No problem - it's fixed. There's another bug in this that needs a little rework, but may well affect all template devices - it's not possible to return `unavailable` in this template.

That should, IMHO, be a new bug and PR in its own right, tho...